### PR TITLE
Add Entity to DB, Rework SME import to DB, Add Parameter to SME-GUI

### PR DESCRIPTION
**Pull request by blidhpxc**
# Description

Add Entity to DB. Rework SME import to DB. Add Parameter to SME-GUI. 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
These changes have been tested by running the application with 5.000 aas and ensuring that all functionalities work as expected.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/AasxServerBlazor/Pages/Db.razor [hunk 1]
```diff
@@ -580,20 +588,19 @@
...
+                                                [@list[ z ][ 1 ]] @list[ z ][ 0 ]
                                             </li>
-                                            z++;
                                         }
                                     </ul>
                                 }
                                 else
                                 {
-                                    @smeDB.getValue()
+                                    @smeDB.getValue().First()[0]
```
Comment (Freezor) [inline src/AasxServerBlazor/Pages/Db.razor:603]: What if .First() returns null? FirstOrDefault would be a better option.

File: src/AasxServerDB/Converter.cs [hunk 2]
```diff
@@ -85,40 +85,35 @@ static private void LoadSME(Submodel submodel, ISubmodelElement? sme, string? SM
             foreach (var smel in smeLevel)
             {
                 ISubmodelElement? nextSME = null;
+                var value = smel.getValue();
                 switch (smel.SMEType)
                 {
                     case "Prop":
-                        nextSME = new Property(DataTypeDefXsd.String, idShort: smel.IdShort, value: smel.getValue());
+                        nextSME = new Property(DataTypeDefXsd.String, value: value.First()[0]);
```
Comment (Freezor) [inline src/AasxServerDB/Converter.cs:92]: Same here. If you use .First and that returns null we get an exception

Comment (blidhpxc) [reply src/AasxServerDB/Converter.cs:92]: The method is designed to always return at least the default value.

Comment (Freezor) [reply src/AasxServerDB/Converter.cs:92]: yes. which is an empty list. And when it is empty and you want to access it first element with `[0]` than there is no element, so you get an exception

Comment (blidhpxc) [reply src/AasxServerDB/Converter.cs:92]: image

Comment (blidhpxc) [reply src/AasxServerDB/Converter.cs:92]: I wanted to avoid always having to check for the first Element because of multiple code and confusing.

Comment (Freezor) [reply src/AasxServerDB/Converter.cs:92]: Can you rebase with the current main. When I try this I get the following issue:

image

**Comment by Freezor** [comment]
@blidhpxc seems to run locally

**Pull request merged**